src/backend/postmaster/postmaster.c

```diff
@@ -1119,7 +1119,7 @@ PostmasterMain(int argc, char *argv[])
 		ereport(ERROR,
 				(errmsg("WAL streaming (max_wal_senders > 0) requires wal_level \"replica\" or \"logical\"")));
 
-    if ( GpIdentity.dbid == -1 && Gp_role == GP_ROLE_UTILITY)
+	if ( GpIdentity.dbid == -1 && IS_UTILITY_OR_SINGLENODE(Gp_role))
```

Comment:
align code.

Reply:
About this line. Actually the original align is not correct. I follow the pattern around this line.